.packit.yaml

```diff
@@ -26,7 +26,8 @@ actions:
     - hatch version
 
 files_to_sync:
-  - fedora/python-requre.spec
```

Comment:
If this is wrong, how come it works in [specfile](https://github.com/packit/specfile/blob/514fbded1f45a665c4a674367719e8811da87659/.packit.yaml#L17-L19)?

BTW, isn't spec file and Packit config synced by default?

Reply:
Not after my changes here https://github.com/packit/packit/pull/2412 -> and this is why I am trying to fix requre for making changes work in my Packit PR.

As I got it, `files_to_sync` was introduced to not implicitly sync files without the user knowing it.
With the introduction of `files_to_sync` every new packit config file has `packit.yaml` and the `specfile` listed in the `files_to_sync` list by default. If a user does not want to sync them he just can remove the files from the list. I am removing not only `synced_files` key but also the flag `sync_default_files` because it was strictly related with the use of `synced_files`.
Every old repo that has not yet added `files_to_sync` has to be updated before the release of packit 1.0.0 or it will not work anymore.

Reply:
Regarding the path I don't know, but I am getting a `packit.exceptions.PackitCommandFailedError: Command 'rsync --archive --ignore-missing-args /tmp/tmpga7e4qw5/fedora/python-requre.spec /tmp/packit-dist-gitt11hupku/fedora/python-requre.spec' failed.` and I think it is related with the missing `fedora` downstream directory.

Reply:
In my PR I have removed the use of `get_all_files_to_sync` related with the `synced_files` key and `sync_default_files` glag. It was this method that was creating the correct mapping between upstream file position and distgit file position...

Reply:
> In my PR I have removed the use of `get_all_files_to_sync` related with the `synced_files` key and `sync_default_files` flag. It was this method that was creating the correct mapping between upstream specfile position and distgit specfile position...

Ok, then it needs to be fixed in specfile as well.

Reply:
> Not after my changes here [packit/packit#2412](https://github.com/packit/packit/pull/2412)

Ok, so there is no default anymore. In that case [ogr](https://github.com/packit/ogr/blob/main/.packit.yaml) should be fixed too.

Reply:
If we should automatically propose PRs to adjust configs I will let specfile and ogr for testing the code we will write.